### PR TITLE
chore: add site metadata

### PR DESCRIPTION
**Pull request by benmccann**
When you Google SvelteKit it shows info from jamstack.org. We should probably have a bit more influence over Google's knowledge graph for this entry

image

### CODE REVIEW

File: sites/kit.svelte.dev/src/routes/+page.svelte [hunk 1]
```diff
@@ -15,6 +15,112 @@
...
+			"startDate": "2022-12-14T17:00:00",
+			"url": [
+				"https://svelte.dev/blog/announcing-sveltekit-1.0",
+				"https://www.youtube.com/watch?v=N4BRVkQVoMc"
+			]
+		},
+		"contributor": [{
+			"@type": "Person",
+			"name": "Rich Harris",
+			"url": "https://www.benmccann.com",
```
Comment (bluwy) [inline sites/kit.svelte.dev/src/routes/+page.svelte:55]: I think Rich's site is at https://www.rich-harris.co.uk (https://github.com/Rich-Harris/Rich-Harris.github.io), looks like it hasn't been updated in a while though 😅 

```suggestion
			"url": "https://www.rich-harris.co.uk",
```

File: sites/kit.svelte.dev/src/routes/+page.svelte [hunk 2]
```diff
@@ -15,6 +15,112 @@
...
+				"https://github.com/Conduitry"
+			]	
+		}, {
+			"@type": "Person",
+			"name": "Dominik Göpel",
+			"url": "https://goepel.it/",
+			"image": "https://avatars.githubusercontent.com/u/611613",
+			"sameAs": [
+				"https://github.com/dominikg"
+			]
```
Comment (bluwy) [inline sites/kit.svelte.dev/src/routes/+page.svelte:108]: Dominik is also recently on https://m.webtoo.ls/@dominikg, cc @dominikg if you'd like to add it

File: sites/kit.svelte.dev/src/routes/+page.svelte [hunk 3]
```diff
@@ -15,6 +15,112 @@
...
+			"url": "https://bjornlu.com/",
+			"image": "https://avatars.githubusercontent.com/u/34116392",
+			"sameAs": [
+				"https://github.com/bluwy",
+				"https://twitter.com/bluwyoo",
+				"https://m.webtoo.ls/@bluwy",
+				"https://www.linkedin.com/in/bjorn-lu-3302a31b5",
+				"https://www.reddit.com/user/IamLUG"
+			]	
+		}]
```
Comment (bluwy) [inline sites/kit.svelte.dev/src/routes/+page.svelte:121]: I've barely contributed to kit directly these days, but thanks for listing me in 😬

File: sites/kit.svelte.dev/src/routes/+page.svelte (hunk 2 above)
Comment (dominikg) [reply sites/kit.svelte.dev/src/routes/+page.svelte:108]:
```suggestion
			"sameAs": [
				"https://github.com/dominikg",
				"https://m.webtoo.ls/@dominikg"
			]	
```

**Comment by Rich-Harris** [comment]
This adds 1kb (zipped, 3kb unzipped) of mostly-useless data to the start of every page load, which seems like a lot. Is that a trade-off we're comfortable with?

**Comment by benmccann** [comment]
It took some finagling, but I managed to cut out 90% of the bytes from the initial page load

**Comment by Rich-Harris** [comment]
Will Google still respect it if it's injected with JS?

**Comment by benmccann** [comment]
Yep: https://developers.google.com/search/docs/appearance/structured-data/generate-structured-data-with-javascript#custom-javascript

**Comment by Rich-Harris** [comment]
This only includes three contributors — what's the criteria for inclusion here? An earlier iteration had a bunch more

**Comment by dummdidumm** [comment]
Is loading this through JS saving anything? Does this have impact on lighthouse which things this is somehow relevant for any metric?

**Comment by benmccann** [comment]
> This only includes three contributors — what's the criteria for inclusion here? An earlier iteration had a bunch more

Oops. I was experimenting with ways to cut down size like removing / reducing parts of the schema info before I figured out the JS trick. It looks like I accidentally used one of these versions instead of the full version. Fixed now. They're in sorted order from https://github.com/sveltejs/kit/graphs/contributors

> Is loading this through JS saving anything? Does this have impact on lighthouse which things this is somehow relevant for any metric?

It allows the initial response from the server to be smaller. The browser can then start parsing and rendering the page a bit sooner. It will only stick the schema info in at the end after showing the page to the user

File: sites/kit.svelte.dev/src/routes/+page.svelte [hunk 4]
```diff
@@ -1,12 +1,22 @@
...
 	import Svelte from './home/Svelte.svelte';
 	import Intro from './home/Intro.svelte';
+	import schema_url from './schema.json?url';
 	import './home/common.css';
+
+	let schema;
+	onMount(async () => {
+		// Google will not allow linking to a schema file. It must be in the DOM
+		// We load it and add it to the DOM to save bytes on page load vs inlining
+		const json = await fetch(schema_url).then((r) => r.text());
```
Comment (benmccann) [inline sites/kit.svelte.dev/src/routes/+page.svelte:17]: @Rich-Harris could this also be written as:
```suggestion
		const json = await (await fetch(schema_url)).text();
```

**Review by Rich-Harris — APPROVED** [review]

**Pull request merged**